ui/apps/platform/src/services/VulnerabilityExceptionService.ts

```diff
@@ -60,13 +60,26 @@ export type VulnerabilityDeferralException = BaseVulnerabilityException & {
     };
 };
 
+export function isDeferralException(
+    exception: BaseVulnerabilityException
+): exception is VulnerabilityDeferralException {
+    return 'targetState' in exception && exception.targetState === 'DEFERRED';
+}
+
 export type VulnerabilityFalsePositiveException = BaseVulnerabilityException & {
+    targetState: 'FALSE_POSITIVE';
     fpRequest: Empty;
```

Comment:
nit: I might have missed something, but why is the field name `fpRequest` and right below we have `falsePositiveUpdate`? I think we should keep it consistent and either say `fp` for both or `falsePositive` for both.

Reply:
Seems like that's how the backend is naming it. @md2119 what was the reason behind the naming convention here? Is this something we should consider changing (even if not right now)?

Reply:
I think it would be nice to elaborate on the field name to make it more intuitive, even if it means a longer name.

Reply:
I'll merge this as-is for now, and when we change field names we can coordinate the FE + BE changes.